test/src/sdk-harness/test_artifacts/tx_contract/src/main.sw

```diff
@@ -3,12 +3,24 @@ contract;
 use std::address::Address;
```

Comment:
I would join all imports from the same place so that the formatter can handle it, including alphabetical ordering (in all files), that aside, does `tx` need to import using `*`?